UpnetixSystemAlertQueue/Classes/AlertFactoryManager.swift

```diff
@@ -139,3 +148,13 @@ open class AlertFactoryManager {
     }
     
 }
+
+public struct TextFieldAttributes {
+    let attributedText: NSAttributedString
```

Comment:
Those two properties could be public also.

Reply:
Also I think we have discussed with @iliyanivanov96 that I believe there should be more properties in this struct to make the textfields more customizable. I think a delegate (UITextFieldDelegate) must be add for sure, it would also be good to have things like keyboardType and returnKeyType too.

Reply:
Fixed